### PR TITLE
Fix issue #220 - Log non-existent settings as warning

### PR DESCRIPTION
**Pull request by charleswhchan**
How does this look?

### CODE REVIEW

File: waffle/models.py [hunk 1]
```diff
@@ -13,6 +14,9 @@
 from waffle import managers
 from waffle.utils import get_setting, keyfmt, get_cache
 
+LOGGER_NAME = 'waffle'
+logger = logging.getLogger(LOGGER_NAME)
```
Comment (jsocol) [inline waffle/models.py]: Can this be a little simpler, just `logger = logging.getLogger('waffle')`? We're not re-using `LOGGER_NAME` anywhere and contextually it's obvious that it's the logger name.

**Review by jsocol — COMMENTED** [review]
one small note but otherwise looks good, thanks!

**Comment by charleswhchan** [comment]
@jsocol: PR updated. Please review again. 😃

**Review by jsocol — APPROVED** [review]

**Comment by jsocol** [comment]
Thank you!

**Comment by charleswhchan** [comment]
Thanks @jsocol 👍

**Pull request merged**